site/docs/recipes/0001-reusable-helpers.md

```diff
@@ -0,0 +1,234 @@
+# Reusable helpers
+
+:::info
+[Here's](https://kyse.link/cPY8e) a playground link containing all the code in this recipe.
+:::
+
+Let's say you want to write the following query:
+
+```sql
+SELECT id, first_name
+FROM person
+WHERE upper(last_name) = $1
+```
+
+Kysely doesn't have a built-in `upper` function but there are at least three ways you could write this:
+
+```ts
+const lastName = 'STALLONE'
+
+const persons = await db
+  .selectFrom('person')
+  .select(['id', 'first_name'])
+  // 1. `sql` template tag. This is the least type-safe option
```

Comment:
```suggestion
  // 1. `sql` template tag. This is the least type-safe option. You're providing the column name without any type-checking.
```

Reply:
Good additions! I'll add these on separate comment lines though to keep the code more readable on mobile.